kk/factories/utils.py

```diff
@@ -1,7 +1,12 @@
 # -*- coding: utf-8 -*-
 
 from django.contrib.auth import get_user_model
+from kk.factories.user import UserFactory
 
 
 def get_random_user():
-    return get_user_model().objects.order_by("?").first()
+    users_qs = get_user_model().objects
+    if not users_qs.count():  # Last-ditch attempt to create _some_ random users
```

Comment:
Yeah. I'm pretty sure no one will attempt to do `get_random_user()` in production code...